src/App.jsx

```diff
@@ -1,104 +1,30 @@
 import './App.scss';
 
-// import postsFromServer from './api/posts.json';
-// import commentsFromServer from './api/comments.json';
-// import usersFromServer from './api/users.json';
+import commentsFromServer from './api/comments.json';
+import postsFromServer from './api/posts.json';
+import usersFromServer from './api/users.json';
```

Comment:
Ensure that the JSON files (`comments.json`, `posts.json`, `users.json`) are correctly structured and contain the expected data. This is crucial for the functions `getUserById` and `getCommentsByPostId` to work correctly.